apps/demo/src/app/home/home.component.html

```diff
@@ -102,6 +102,25 @@ <h2 class="title">
     </div>
 </div>
 
+<div class="wrapper">
```

Comment:
Should it only be visible on mobile devices?

Reply:
fixed